### PR TITLE
Feat/filter checkbox component

### PR DESCRIPTION
**Pull request by dariobk**

### CODE REVIEW

File: src/_includes/filter-checkbox.liquid [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<div><input type="checkbox" class="filter-checkbox__input" id="{{ isReference }}" name="{{ name }}"/>
```
Comment (s2-schwaddi) [inline src/_includes/filter-checkbox.liquid:1]: please add another property for `value`

File: src/styles/includes/_filter-checkbox.scss [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+	background-color: $color-primary--beige;
+	display: flex;
+	height: 3.6rem;
+	cursor: pointer;
+
+	.is-dark & {
+		background-color: $color-primary--darkgray;
+	}
+
+	&__input {
```
Comment (s2-schwaddi) [inline src/styles/includes/_filter-checkbox.scss:15]: @trinegl does that fulfill accessibility requirements? Will it be recognised as an input field on assistive techologies?

File: src/styles/includes/_filter-checkbox.scss [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+		appearance: none;
+	}
+
+	&__text {
+		margin-right: $spacing--200;
+		font-size: $font-size--copy-2;
+		font-family: $font-sans-serif-semibold;
+		line-height: $line-height--copy-2;
+		letter-spacing: 0.15rem;
```
Comment (s2-schwaddi) [inline src/styles/includes/_filter-checkbox.scss:24]: Consider to put this into variables

File: src/styles/includes/_filter-checkbox.scss (hunk 2 above)
Comment (trinegl) [reply src/styles/includes/_filter-checkbox.scss:15]: Yes, this should be fine, as the HTML element using this class is actually an input element.

File: src/styles/includes/_filter-checkbox.scss [hunk 4]
```diff
@@ -0,0 +1,107 @@
+@import "../core/variables";
+@import "../core/mixins";
+
+.filter-checkbox {
+	border-radius: 10rem;
```
Comment (timbru31) [inline src/styles/includes/_filter-checkbox.scss:5]: We could think to make this a variable, as this is used quite some times. Can be done in a new PR though.

**Review by timbru31 — APPROVED** [review]

**Pull request merged**